src/mantine-dates/src/components/TimeRangeInput/TimeRangeInput.story.tsx

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-console */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react';
+import { MANTINE_SIZES } from '@mantine/core';
+import { TimeRangeInput } from './TimeRangeInput';
+
+const sizes = MANTINE_SIZES.map((size) => (
+  <TimeRangeInput label={size} size={size} key={size} style={{ marginTop: 30 }} />
+));
+
+function Controlled() {
+  const [value, setValue] = useState([new Date(), new Date()]);
+  const onChange = (val) => {
+    setValue(val);
+  };
+  return <TimeRangeInput value={[value[0], value[1]]} onChange={onChange} label="Controlled" withSeconds />;
+}
+
+storiesOf('@mantine/dates/TimeRangeInput', module)
+  .add('General usage', () => (
+    <TimeRangeInput
+      label="Time Range input"
+      disabled
+      styles={{
+      wrapper: {
```

Comment:
Please make sure that you have prettier installed in your code editor, there are lots formatting issues that can be automatically fixed